fixing the prijsherzieningspost

@claude the blogpost is not appearing on the website: 1s
1s
Run set -e
Downloading from: https://economie.fgov.be/sites/default/files/Files/Entreprises/prix-construction-Indice-I-2021.xlsx
Downloaded 57665 bytes to /home/runner/work/data-blog/data-blog/embuild-analyses/analyses/prijsherziening-index-i-2021/data/prix-construction-Indice-I-2021.xlsx
Excel sheets: ['Taalkeuze-Choix de langue', 'I_2021 (Fr)', 'I_2021 (Nl)']
Loaded 5 rows, 3 columns
Columns: ['Unnamed: 0', 'Unnamed: 1', 'Unnamed: 2']
Traceback (most recent call last):
  File "/home/runner/work/data-blog/data-blog/embuild-analyses/analyses/prijsherziening-index-i-2021/src/process_prijsherziening.py", line 187, in <module>
    process_data(excel_path)
    ~~~~~~~~~~~~^^^^^^^^^^^^
  File "/home/runner/work/data-blog/data-blog/embuild-analyses/analyses/prijsherziening-index-i-2021/src/process_prijsherziening.py", line 149, in process_data
    df_pivot = df_export.pivot_table(
               ~~~~~~~~~~~~~~~~~~~~~^
        index=['year', 'month'],
        ^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
        aggfunc='first'
        ^^^^^^^^^^^^^^^
    ).reset_index()
    ^
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/pandas/core/frame.py", line 9536, in pivot_table
    return pivot_table(
        self,
    ...<9 lines>...
        sort=sort,
    )
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/pandas/core/reshape/pivot.py", line 102, in pivot_table
    table = __internal_pivot_table(
        data,
    ...<9 lines>...
        sort,
    )
  File "/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/pandas/core/reshape/pivot.py", line 148, in __internal_pivot_table
    raise KeyError(i)
KeyError: 'value'
Error: Process completed with exit code 1.

--- a/embuild-analyses/analyses/prijsherziening-index-i-2021/src/process_prijsherziening.py
+++ b/embuild-analyses/analyses/prijsherziening-index-i-2021/src/process_prijsherziening.py
@@ -80,8 +80,8 @@ def process_data(excel_path: str) -> None:
     xl_file = pd.ExcelFile(excel_path)
     print(f"Excel sheets: {xl_file.sheet_names}")
 
-    # Read the first sheet (assuming it contains the data)
-    df = pd.read_excel(excel_path, sheet_name=0)
+    # Read the Dutch data sheet (I_2021 (Nl))
+    df = pd.read_excel(excel_path, sheet_name='I_2021 (Nl)')
     print(f"Loaded {len(df)} rows, {len(df.columns)} columns")
     print(f"Columns: {df.columns.tolist()}")
 

